CHANGES.rst

```diff
@@ -4,7 +4,8 @@ Changelog
 1.3.1 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Turn fucntional tests into better covering unit tests and also add more tests.
```

Comment:
Typo: fucntional -> functional.

I also think "better-covering" needs a hyphen, but I'm not a native English speaker and may be mistaken.